### DISCUSSION TITLE
Folder-specific settings ignored after restart?

### DISCUSSION THREAD

**Discussion by diwasrimal** [zed-industries/zed#12633] [Feature Requests]
I am working on a React Typescript project which used 2 spaces for indentation by default. Since I prefer 4, I made a `.zed/settings.json` file
```json
// Folder-specific settings
//
// For a full list of overridable settings, and general information on folder-specific settings,
// see the documentation: https://zed.dev/docs/configuring-zed#folder-specific-settings
{
    "tab_size": 4
}
```

When I quit Zed (cmd+q), and reopen, the tab size is set to 2 again. I have to open `.zed/settings.json` again to use local settings. But If I had `.zed/settings.json` file previously opened in a buffer, the settings are applied.

https://github.com/zed-industries/zed/assets/84910758/c44ec3d1-51f1-4c62-96a7-257e432b6053

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
